webpack.config.js

```diff
@@ -0,0 +1,36 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = function(fabricatorConfig) {
+
+	var config = {
+		entry: {
+			'fabricator/scripts/f': fabricatorConfig.src.scripts.fabricator,
```

Comment:
I removed the `scripts:fabricator` task in favor of just bundling it with weback.